decidim-core/app/models/decidim/authorization.rb

```diff
@@ -71,5 +85,13 @@ def active_handler?
     def workflow_manifest
       @workflow_manifest ||= Decidim::Verifications.find_workflow_manifest(name)
     end
+
+    # Calculates when this authorization can be reseted, if desired.
+    #
+    # time_between_renewals defaults to zero
```

Comment:
I've mixed both suggestions because, if it's enabled and not defined in the workflow you get the default value:

```rb
# **time_between_renewals** can be defined in `workflow_manifest.time_between_renewals` 
# defaults to 1 day
```